<launch>
<node name="ros_corgi_bridge" pkg="quadruped" type="ros_corgi_bridge" output="screen"/> 
</launch>
